test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs

```diff
@@ -355,8 +355,11 @@ public RazorPagesTest(MvcTestFixture<RazorPagesWebSite.Startup> fixture)
             Assert.Equal("Hello, handler!", content.Trim());
         }
 
-        [Fact]
-        public async Task HelloWorldWithPageModelHandler_CanPostContent()
+        [Theory]
+        [InlineData("POST")]
+        [InlineData("poST")]
```

Comment:
Do we really need functional tests that vary in case? Unit tests should suffice for this.

Reply:
It's a fair cop